.vscode/tasks.json

```diff
@@ -16,9 +16,7 @@
       "label": "Pytest",
       "type": "shell",
       "command": "pytest --timeout=10 tests",
-      "dependsOn": [
-        "Install all Test Requirements"
-      ],
+      "dependsOn": ["Install all Test Requirements"],
```

Comment:
Not sure why precommit fixed this, happy to revert, but perhaps this should just be formatted differently nowadays :-).